tools/get-keys-from-ts.js

```diff
@@ -260,7 +260,7 @@ function getKeysFromTs(code, {
     const initialNodes = {};
 
     /**
-     * Finds a TypeScript interfaction declaration.
+     * Finds a TypeScript interaction declaration.
```

Comment:
[interfaction](https://en.wiktionary.org/wiki/interfaction) is a valid word

Reply:
I can't find anything that indicates it's intended: https://www.google.com/search?sxsrf=APwXEdcQwHkJy68lJ42fE3U3bPNyjsnFlQ:1685414019833&q=%22TypeScript+interfaction%22&nfpr=1&sa=X&ved=2ahUKEwix9M2_gJz_AhUFlYkEHcVWBOoQvgUoAXoECA4QAg&biw=948&bih=1000&dpr=2

Reply:
Looking more closely, I think the intended word was actually `interface`...

Reply:
Yes, `interface`.